pip/index.py

```diff
@@ -1194,6 +1194,10 @@ def verifiable(self):
             # This link came from an untrusted source and we cannot trust it
             return False
 
+    @property
+    def is_wheel(self):
```

Comment:
Well, the Link class is already full of properties so it seems more coherent to make it a property.